map_test.go

```diff
@@ -144,6 +144,61 @@ func createArray(t *testing.T) *Map {
 	return m
 }
 
+func TestMapQueue(t *testing.T) {
+
+	testutils.SkipOnOldKernel(t, "4.20", "map type queue")
```

Comment:
the map type queue and stack was added in [1]. So this feature is not available for older kernel.

[1] https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/include/uapi/linux/bpf.h?id=f1a2e44a3aeccb3ff18d3ccc0b0203e70b95bd92

Reply:
Sounds good, thanks for checking.